core/templates/dev/head/components/share/sharing_links_directive.html

```diff
@@ -1,32 +1,35 @@
 <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
   <li>
     <a ng-href="https://plus.google.com/share?url=<[serverName]>/<[activityType]>/<[activityId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" ng-click="registerShareEvent('gplus')" target="_window">
-      <img class="share-option-img" ng-src="<[gplusUrl]>" alt="Google+">
+      <i class="oppia-share-icons fa fa-google-plus-square"></i>
+      <span class="oppia-icon-accessibility-label">Google+</span>
     </a>
   </li>
 
   <li>
     <a ng-href="https://www.facebook.com/sharer/sharer.php?sdk=joey&u=<[serverName]>/<[activityType]>/<[activityId]>&display=popup&ref=plugin&src=share_button" onclick="return !window.open(this.href, '', 'height=336, width=640')" ng-click="registerShareEvent('facebook')" target="_window">
-      <img class="share-option-img" ng-src="<[fbUrl]>" alt="Facebook">
+      <i class="oppia-share-icons fa fa-facebook"></i>
+      <span class="oppia-icon-accessibility-label">Facebook</span>
     </a>
   </li>
 
   <li>
     <a ng-href="https://twitter.com/share?text=<[escapedTwitterText]>&url=<[serverName]>/<[activityType]>/<[activityId]>" onclick="return !window.open(this.href, '', 'height=460, width=640')" ng-click="registerShareEvent('twitter')" target="_window">
-      <img class="share-option-img" ng-src="<[twitterUrl]>" alt="Twitter">
+      <i class="oppia-share-icons fa fa-twitter"></i>
+      <span class="oppia-icon-accessibility-label">Twitter</span>    
     </a>
   </li>
 
   <li>
     <a ng-href="https://classroom.google.com/share?url=<[serverName]>/<[activityType]>/<[activityId]>" onclick="return !window.open(this.href, '', 'height=460, width=640')" ng-click="registerShareEvent('classroom')" target="_window">
-      <img class="share-option-img" ng-src="<[classroomUrl]>" alt="Classroom">
+      <img class="share-option-img" ng-src="<[classroomUrl]>" alt="Classroom">    
```

Comment:
I am +1 for updating to font-awesome but not in this PR as few icons changes over versions and can result in unexpected behaviour over all font-awesome icons all over the site.

Reply:
Updated before + after icons screenshots

Reply:
Ah, thanks!

Yes, separate PR for that is fine.

Looking at the screenshots, there is a noticeable difference in the consistency of the icons before-and-after. Could we still mimic the look-and-feel of what it was before, but using font-awesome icons on a square background of suitable colour?

Reply:
Yes, sure. I can use squared version of those

Reply:
Hi @seanlip, I think we already have a check for trailing whitespaces. I will check this once and update it here. Thanks!

Reply:
@apb7 No worries. I manually passed the pre-hook file as my defaut configuration had python3 which is not compatible with `pre-hook.py`. Also, can you guys please see to it that all scripts are compatible with python3 as thats becoming new standard among community. Thanks!

Reply:
I ran this locally. We do have a check for this. This is the error message:
```bash
Starting HTML linter...
----------------------------------------

Linting /home/apurv/dev/oppia/./core/templates/dev/head/components/share/sharing_links_directive.html file
/home/apurv/dev/oppia/core/templates/dev/head/components/share/sharing_links_directive.html: line 22, col 78, line contains trailing whitespace

[htmllint] found 1 errors out of 1 files

----------------------------------------
(1 files checked, 1 errors found)

FAILED   HTML linting failed
HTML linting finished.
```
Also, @shaz13 we support Python 2.7 as of now. The complete code base uses it.
Thanks!

Reply:
Ah ok, @apb7, thanks for checking!

@shaz13 I think we need to get the presubmit checks working on your machine without any trouble, otherwise all this stuff is going to come up again and again during code reviews. Could you please work with @apb7 to get it working smoothly? Anything you push to GitHub should go through these checks.

Thanks!

Reply:
Check seems to pass now :+1: PTAL